cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureCredentialAppCreationCommand.java

```diff
@@ -88,8 +88,8 @@ public AzureApplicationCreationView generateJSON(String deploymentAddress) {
 
     String getRedirectURL(String workspaceId, String deploymentAddress) {
         String cbAzAppAuthUri = String.format(CB_AZ_APP_REDIRECT_URI_PATTERN, workspaceId);
-        String replyUrl = deploymentAddress.endsWith(DELIMITER) ? deploymentAddress : deploymentAddress.concat(DELIMITER);
-        return replyUrl.concat(cbAzAppAuthUri);
+        String replyUrl = deploymentAddress.endsWith(DELIMITER) ? deploymentAddress : deploymentAddress + DELIMITER;
+        return replyUrl + cbAzAppAuthUri;
```

Comment:
what is the problem with _concat()_

Reply:
Simply unnecessary, concat is an explicit method call meanwhile + is optimized by the compiler